ome-demoserver.yml

```diff
@@ -119,15 +119,15 @@
         databases: ["{{ vault.omero_server_dbname }}"]
 
     - role: openmicroscopy.omero-server
-      omero_server_release: 5.4.5
+      omero_server_release: "{{ omero_server_release_desired | default('5.4.5') }}"
       omero_server_dbuser: "{{ vault.omero_server_db_user }}"
       omero_server_dbname:  "{{ vault.omero_server_dbname }}"
       omero_server_dbpassword: "{{ vault.omero_server_dbpassword }}"
       omero_server_rootpassword: "{{ vault.omero_server_rootpassword }}"
       omero_server_systemd_limit_nofile: 16384
 
     - role: openmicroscopy.omero-web
-      omero_web_release: 5.4.5
+      omero_web_release: "{{ omero_server_release_desired | default('5.4.5') }}"
```

Comment:
`s/omero_server_release_desired/omero_web_release_desired`?

Reply:
d'oh - thanks. Updated and pushed.